workflow-controller/controller-service/src/main/java/io/hyscale/controller/manager/impl/LocalRegistryManagerImpl.java

```diff
@@ -82,7 +83,9 @@ public ImageRegistry getImageRegistry(String registry) throws HyscaleException {
     public ImageRegistry getImageRegistry(DockerConfig dockerConfig, String registry) {
 
         if (StringUtils.isBlank(registry)) {
-            return null;
+        	ImageRegistry imageRegistry=new ImageRegistry();
```

Comment:
This image registry object will also be used to create image pull secret. For that other details need to be populated. If registry is blank we should set it as default one here and continue with the flow.

Reply:
If registry is blank then get Image registry by dockerhubaliases